### DISCUSSION TITLE
How can see all registered clients

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#870] [General]
Dear Sirs, I have installed self host  ID/rendezvous & relay server and I have a question: How can see all the registered clients to my server? Is there any way to delete or block a client?
Thanks a lot.

**Comment by rustdesk** [comment]
no way for the time being.

**Comment by mikekgr** [comment]
I see, do you have plans to implement it?
Thanks.

**Comment by rustdesk** [comment]
Yes, pro version.

**Comment by mikekgr** [comment]
Very nice. Is it ready made the pro version or still under development ?  Have you decided about the cost of it?

**Comment by rustdesk** [comment]
Not yet. We focus on polishing client for now, pro version won't be launched soon.

**Comment by supply9243** [comment]
You can view their ID, registration time, and registration IP through the db_v2.sqlite3 file. You can use the server to add the "-k _" parameter to prevent client connections without the same public key.This is only a temporary method

**Reply by mikekgr** [reply]
Thanks. Please show me the way (commands from cli , linux / Ubuntu server) to get these info's from db_v2.sqlite3 I am not sqlite3 experienced at all.

**Reply by supply9243** [reply]
https://github.com/sqlitebrowser/sqlitebrowser
or other similar software

**Reply by mikekgr** [reply]
Perfect. Thanks a lot !
